database/menuSeeding.js

```diff
@@ -0,0 +1,53 @@
+const db = require('./index.js');
+const Menu = require('./Menu.js');
+const faker = require('faker');
+var generateDishNameAndPrice = function() {
```

Comment:
consider using ES6 arrow function, as well as let/const over var. eslint also highlights everywhere that needs a semi-colon, stylistically